src/charm.py

```diff
@@ -51,6 +53,33 @@ def _on_config_changed(self, _: ops.ConfigChangedEvent) -> None:
         """Config changed handler."""
         self.reconcile()
 
+    def _on_install(self, _: ops.InstallEvent) -> None:
+        """Install event handler."""
+        if not self.state:
+            return
+        # This is the default user password
+        default_token = "Bearer wazuh:wazuh"  # nosec
+        try:
+            r = requests.put(
+                "https://localhost:55000/security/users/2",
```

Comment:
Does this mean that users ID 1 and 2 will have the same password?

Reply:
Yes, I'm still not sure what's the use for user 2 but it's also a system user. User 1 is the API's admin user. I've also considered setting a random password for user 2. WDYT?

Reply:
Updated user 2 password to a random value. I don't expect to use it.